### ISSUE TITLE
csharp_space_between_method_call_parameter_list_parentheses is not honored in variable initialization

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/roslyn#43791]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/content/problem/1011050/csharp-space-between-method-call-parameter-list-pa.html)._

---
In VS 2019, I went to Tools -> Options -> Text Editor -> C# -> Code Style -> Formatting -> Spacing; and I enabled the option "Insert space within argument list parentheses", which corresponds to the .editorconfig option `csharp_space_between_method_call_parameter_list_parentheses`.

I generated a .editorconfig file based on this using VS and saved it to the root of my solution.

After I pressed the Code Cleanup button, the new style was applied, except as follows:

### Before:

```
            byte[] codePointBytes = Encoding.UTF32.GetBytes(s);
            codePointBytes = Encoding.UTF32.GetBytes(s);
            Encoding.UTF32.GetBytes(s);
```

### Expected result:

```
            byte[] codePointBytes = Encoding.UTF32.GetBytes( s );
            codePointBytes = Encoding.UTF32.GetBytes( s );
            Encoding.UTF32.GetBytes( s );
```

### Actual result:

```
            byte[] codePointBytes = Encoding.UTF32.GetBytes(s);    // Space was not added inside parentheses
            codePointBytes = Encoding.UTF32.GetBytes( s );
            Encoding.UTF32.GetBytes( s );
```

To summarize, VS does not apply the code style `csharp_space_between_method_call_parameter_list_parentheses` when the method call is part of a variable initialization.

---
#### Visual Studio Feedback System on 4/29/2020, 02:13 AM: 

<p>We have directed your feedback to the appropriate engineering team for further evaluation. The team will review the feedback and notify you about the next steps.</p>

---
### Original Solutions
(no solutions)

**Comment by MMS1980** [comment]
Please ensure
csharp_space_around_declaration_statements=false
if it is "ignore" csharp_space_between_method_call_parameter_list_parentheses will have no effect on declarations.
